lib/livebook/runtime/evaluator/doctests.ex

```diff
@@ -36,7 +36,18 @@ defmodule Livebook.Runtime.Evaluator.Doctests do
   end
 
   defp define_test_module(modules) do
-    name = Module.concat([LivebookDoctest | modules])
+    id =
+      modules
+      |> Enum.sort()
+      |> Enum.map_join("-", fn module ->
+        module
+        |> Atom.to_string()
+        |> String.replace_prefix("Elixir.", "")
+      end)
+      |> :erlang.md5()
+      |> Base.encode64()
```

Comment:
```suggestion
      |> Base.encode32(padding: false)
```

So the module names are still valid?

Reply:
Ah good call!